examples/shaders/shaders_basic_pbr.c

```diff
@@ -122,7 +123,7 @@ int main()
     SetShaderValue(shader, GetShaderLocation(shader, "ambientColor"), &ambientColorNormalized, SHADER_UNIFORM_VEC3);
     SetShaderValue(shader, GetShaderLocation(shader, "ambient"), &ambientIntensity, SHADER_UNIFORM_FLOAT);
 
-    // Get location for shader parameters that can be modified in real time
+    // Get the location for shader parameters that can be modified in real time
```

Comment:
I'm afraid I'm not using `the` in any of the shaders examples, for consitency, I prefer to keep it without it.